Packs/Akamai_SIEM/Integrations/Akamai_SIEM/Akamai_SIEM.py

```diff
@@ -379,6 +474,16 @@ def main():
                                                               last_run=demisto.getLastRun().get('lastRun'))
             demisto.incidents(incidents)
             demisto.setLastRun(new_last_run)
+        elif command == "fetch-events":
+            for events, offset in fetch_events_command(
+                client,
+                params.get("fetchTime"),
+                params.get("fetchLimit"),
+                params.get("configIds"),
+                ctx=get_integration_context() or {},
+            ):
+                send_events_to_xsiam(events, VENDOR, PRODUCT)
+                set_integration_context({"offset": offset})
```

Comment:
Since we split into packets, we want to be able to keep track in case the instance crushed in the middle of a fetch cycle.
I.e:
If the packet size is 50k, the limit is 200k, and we fetched 100k and then crushed, we want to ensure we will continue from where we crushed.

Reply:
if thats the case lets do the updatemodulehealth only after we sent all the events. lets discuss